lib/experimental/Widgets/WidgetSection/index.tsx

```diff
@@ -0,0 +1,23 @@
+import { useTextFormatEnforcer } from "@/lib/text"
+import { forwardRef, PropsWithChildren } from "react"
+
+type Props = PropsWithChildren & {
+  title?: string
+}
+
+export const WidgetSection = forwardRef<HTMLDivElement, Props>(
+  ({ title, children }, ref) => {
+    useTextFormatEnforcer(title, { disallowEmpty: true })
+
+    return (
+      <div ref={ref} className="flex flex-col gap-2">
+        <p className="text-base font-medium text-f1-foreground-secondary">
```

Comment:
shouldn't it be `h3` or `h4`?

Reply:
Yep, it's better semantically, done!

Reply:
Actually we cannot add a `h4` and some `className` at the same time, it's detected as an accesibility violation